src/havard_eig_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,18 @@
+def bubble_sort(dataa):
+    data_list = list(dataa)
+    for i in range(len(data_list) - 1, 0, -1):
+        for j in range(i):
+            if data_list[j] > data_list[j + 1]:
+                replace_temp = data_list[j]
+                data_list[j] = data_list[j + 1]
+                data_list[j + 1] = replace_temp
```

Comment:
Python swapping is very easy:
`i, j = j, i` 
You don't need a `temp` variable here.